doc/snapper.xml.in

```diff
@@ -89,6 +89,8 @@
       <para>With each snapshot a description and some userdata can be associated. The
       description is a string. The userdata is a list of key-value pairs where the
       keys and values are strings.</para>
+      <para>Do not use non-ASCII characters for the snapshot description, unless you
+      always use the UTF-8 character encoding.</para>
```

Comment:
This also applies to userdata and cleanup algorithm. Also most table formats do only work with UTF-8.

Reply:
Thanks, adjusted the text (to cover other strings too).